### DISCUSSION TITLE
Nomad integration

### DISCUSSION THREAD

**Discussion by wizpresso-steve-cy-fan** [coder/coder#10782] [General] [answered]
We should have an official guide on integration with Nomad rather than just Kubernetes.

**Comment by matifali** [comment, marked as answer]
Hi @wizpresso-steve-cy-fan, Thank you for your interest. 

Are you looking to deploy Coder on Nomad or just Nomad workspaces? We publish an official [Nomad starter template](https://registry.coder.com/templates/nomad-docker) that allows creating workspaces on Nomad.
